app/views/shared/buttons/_ab_options_form.html.haml

```diff
@@ -69,14 +69,15 @@
                         options_for_select([[_("<None>"), nil]] + Array(@edit[:new][:available_dialogs].invert).sort_by { |a| a.first.downcase }, @edit[:new][:dialog_id]),
                         "data-miq_sparkle_on" => true,
                         :class => "selectpicker")
-    .form-group
-      %label.control-label.col-md-2
-        = _('Open URL')
-      .col-md-8
-        = check_box_tag("open_url", "1", @edit[:new][:open_url], "data-miq_observe_checkbox" => {:interval => '.5', :url => url}.to_json, disabled: @edit[:new][:disabled_open_url])
-        - if @edit[:new][:disabled_open_url]
-          .note
-            = _("Only available for VM with \"Display for\" set to \"Single\"")
+    - if @resolve[:target_class] == "VM and Instance"
```

Comment:
This isn't going to work, because content of `@resolve[:target_class]` will be translated into currently used locale.

You need to use more robust mechanism here, something that doesn't depend on an ad-hoc string.

Reply:
thanks, removed that part